lib/flow/Advection.cpp

```diff
@@ -387,8 +387,10 @@ Advection::OutputStreamsGnuplotMaxPart( const std::string&  filename,
     if( f == nullptr )
         return FILE_ERROR;
 
+    int idx = 0;
     for( const auto& s : _streams )
     {
+        std::fprintf( f, "# Flow line index = %d\n", idx++ );
```

Comment:
@shaomeng I've been thinking that what might be most helpful is to output the file in CSV format so that it can be directly imported in a meaningful way by applications such as excel and pandas that can import CSV files. I think it is ok to have a comment at the top, but then the "header" and values should follow the CSV formant. So instead of:

```
# X-position      Y-position      Z-position     Time     Value

# Flow line index = 9
-377109.875000, 2508461.500000, 31.711678, 146966400.000000, 22.783976
-373682.562500, 2510747.000000, 31.711678, 146966576.000000, 23.498034
```
we'd have:

```
ID, X-position, Y-position, Z-position, Time, Value
0, -377109.875000, 2508461.500000, 31.711678, 146966400.000000, 22.783976
0, -373682.562500, 2510747.000000, 31.711678, 146966576.000000, 23.498034

```
What do you think?